pages/common/neofetch

```diff
@@ -0,0 +1,24 @@
+# neofetch
+
+> CLI system tool to display information about your system next to an image, your OS logo, or any ASCII file of your choice.
+> NOTE: Every launch flag has a config option.
+
+- Call neofetch (if the script config file has been set up previously):
+
+`neofetch`
+
+- Pass specific parameters:
+
+`neofetch --option "value" --option "value"`
+
+- Disable an info line from appearing in the output:
+
+`neofetch --disable infoname`
+
+- Hide/Show OS architecture:
+
+`neofetch --os_arch on/off`
```

Comment:
Add token syntax around `on/off` and separate with a pipe instead of a slash.

Like: `{{on|off}}`